raritan/logger.py

```diff
@@ -1,101 +1,71 @@
-import logging
 
 from rich.console import Console
-from rich.logging import RichHandler
-from rich.theme import Theme
 
 from raritan.context import context
 
 """
-Provides logging utilities.
+Some very simple rich enabled CLI output helpers.
 """
 
-# We are generally only ever going to be operating in a logging or no logging state.
-# Store and make static, a configured logger for our current run mode.
-active_logger = None
+# A logger instance for use here and elsewhere.
```

Comment:
Did you consider using a plugin for logging? I'm just curious why you chose to roll your own logger.

Reply:
We originally used the core python core logger module. For now it was overkill for what we need and it actually made testing ETL output much harder. I rolled my own because adding a tiny bit of custom code felt better than maintaining an external dependency.